mobile-test-app/package.json

```diff
@@ -7,9 +7,9 @@
     "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
-    "@ledgerhq/errors": "^4.71.0",
-    "@ledgerhq/live-common": "^8.0.0-alpha.2",
-    "@ledgerhq/react-native-ledger-core": "^3.2.1",
+    "@ledgerhq/errors": "^4.72.1",
+    "@ledgerhq/live-common": "file:.yalc/@ledgerhq/live-common",
```

Comment:
watch out if you merge ^^

Reply:
It's ok because need to do a release first